examples/storybook/src/wrap-with-hits.ts

```diff
@@ -24,94 +24,41 @@ export function wrapWithHits({
     selector: 'ais-app',
     template: `
       <ais-instantsearch [config]="config">
-        <div id="widget-display">
+        <div class="ais-container ais-container-preview">
           ${template}
         </div>
-        <div id="results-display">
-          <div id="results-search-box-container">
-            <ais-search-box
-              placeholder="Search into furnitures"
-            >
-            </ais-search-box>
-          </div>
-          <div id="results-hits-container">
+        <div class="ais-container ais-container-playground">
```

Comment:
Would be nice to have the same kind of layout than Vue that adds a refinement widget by default (like a `Menu` or `RefinementList)`. It's easier to test widgets like `ClearRefinements` & `CurrentRefinements`. We can use a default value and let the story override the value if needed (see [Vue](https://github.com/algolia/vue-instantsearch/blob/v2/stories/utils.js#L30-L32)).

Reply:
done!